src/Stomp/Network/Connection.php

```diff
@@ -454,4 +479,17 @@ public function getHost()
     {
         return $this->host;
     }
+
+    /**
+     * Writes an "alive" message on the connection to indicate that the client is alive.
+     *
+     * @return bool
+     */
+    public function sendAlive()
+    {
+        if ($this->isConnected()) {
+            return (!@fwrite($this->connection, "\n", 1) == true);
+        }
```

Comment:
It's easy to see that this might return nothing good...
Good reason to wait until Tuesday before opening a PR the next time ;)